cob_bringup/robots/raw3-1-calib.launch

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0"?>
+<launch>
```

Comment:
The calib can be removed, this was just a temporary setting of my student which we did not mean to add to github.